cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICKANSinkTest.java

```diff
@@ -43,13 +44,182 @@ public NGSICKANSinkTest() {
     } // NGSICKANSinkTest
 
     /**
+     * [NGSICKANSink.configure] -------- When not configured, not mandatory parameters get default values.
+     */
+    @Test
+    public void testConfigureDefaults() {
+        System.out.println(getTestTraceHead("[NGSICKANSink.configure]")
+                + "-------- When not configured, not mandatory parameters get default values");
+        String apiKey = null; // default
+        String attrPersistence = null; // default
+        String backendMaxConns = null; // default
+        String backendMaxConnsPerRoute = null; // default
+        String batchSize = null; // default
+        String batchTime = null; // default
+        String batchTTL = null; // default
+        String dataModel = null; // default
+        String enableEncoding = null; // default
+        String enableGrouping = null; // default
+        String enableLowercase = null; // default
+        String host = null; // default
+        String port = null; // default
+        String ssl = null; // default
+        NGSICKANSink sink = new NGSICKANSink();
+        sink.configure(createContext(apiKey, attrPersistence, backendMaxConns, backendMaxConnsPerRoute, batchSize,
+                batchTime, batchTTL, dataModel, enableEncoding, enableGrouping, enableLowercase, host, port, ssl));
+        
+        try {
+            assertTrue(sink.getRowAttrPersistence());
+            System.out.println(getTestTraceHead("[NGSICKANSink.configure]")
+                    + "-  OK  - 'attr_persisnce=row' configured by default");
+        } catch (AssertionError e) {
+            System.out.println(getTestTraceHead("[NGSICKANSink.configure]")
+                    + "- FAIL - 'attr_persisnce=row' not configured by default");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1162/commits/e8604df05d9f9d9033f2810069d8c8cc76633587